docs/next/guides/upgrade-and-migration/migrating-from-11.1-to-12.0.md

```diff
@@ -12,3 +12,116 @@ pageClass: migration-guide
+| <span style="white-space: nowrap;">[`updateSettings()`](@/api/core.md#updatesettings)</span><br> with [`data`](@/api/options.md#data) defined | <ul><li>Always replaces [`data`](@/api/options.md#data) using [`loadData()`](@/api/core.md#loaddata)</li><li>[Options](@/guides/getting-started/setting-options.md) are reset</li><li>Index mappings are reset</li></ul>                             | <ul><li>At initialization, replaces [`data`](@/api/options.md#data) using [`loadData()`](@/api/core.md#loaddata)</li><li>After initialization, replaces [`data`](@/api/options.md#data) using [`updateData()`](@/api/core.md#updatedata)</li><li>[Options](@/guides/getting-started/setting-options.md) are not reset</li><li>Index mappings are not reset</li></ul> |
+| <span style="white-space: nowrap;">[`loadData()`](@/api/core.md#loaddata)</span>                                                              | Hooks triggered:<ul><li>[`beforeLoadData`](@/api/hooks.md#beforeloaddata)</li><li>[`afterLoadData`](@/api/hooks.md#afterloaddata)</li><li>[`afterChange`](@/api/hooks.md#afterchange), with the `source` argument declared as `loadData()`</li></ul> | Same as before                                                                                                                                                                                                                                                                                                                                                       |
+| <span style="white-space: nowrap;">[`updateData()`](@/api/core.md#updatedata)</span>                                                          | N/A                                                                                                                                                                                                                                                  | Hooks triggered:<ul><li>[`beforeLoadData`](@/api/hooks.md#beforeloaddata)</li><li>[`afterLoadData`](@/api/hooks.md#afterloaddata)</li><li>[`afterChange`](@/api/hooks.md#afterchange), with the `source` argument declared as `updateData()`</li></ul>                                                                                                               |
```

Comment:
I don't understand the rows about `loadData` and `updateData` (lines 32-33) in this migration guide. I thought there are no changes regarding these methods in 12.0.0. All changes were made in 11.1.0.

@jansiegel can you comment on this?

Screenshot, because the Markdown is not very readable here:

<img width="920" alt="Screen Shot 2022-03-09 at 6 11 46 PM" src="https://user-images.githubusercontent.com/566463/157494335-df8ce99b-c77e-4d80-b9e7-9d16e7807029.png">

Reply:
@warpech  @kirszenbaum  True, you're right, I confused those two changes.

Reply:
Ok, I thought we're just listing them to show a bigger picture. Updated: [e92ee45](https://github.com/handsontable/handsontable/pull/9271/commits/e92ee45ad605d1cd9c0f378e571cd27ff188a8a4)

<img width="837" alt="1" src="https://user-images.githubusercontent.com/29247866/157620858-9415f4a4-fdc4-491d-b963-3dae88e609b5.png">